src/dcp/dcp.c

```diff
@@ -518,8 +525,8 @@ int main(int argc, char** argv)
         /* destroy snapshot after copy */
         if (rank == 0) {
             daos_epoch_range_t epr;
-            epr.epr_lo = epoch;
-            epr.epr_hi = epoch;
+            epr.epr_lo = daos_args->epc;
+            epr.epr_hi = daos_args->epc;
```

Comment:
@daltonbohning this was my mistake, but we should add an if here to only destroy if the snapshot wasn't passed in by the user. I had to go back and add this to the daos utility myself after you brought it up.